src/main/java/seedu/address/ui/Tab.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.ui;
+
+/**
+ * Existing tabs.
+ */
+public enum Tab {
+    CLASSES(0),
+    ATTENDANCE(1);
+
+    /** Integer to represent the order of the tabs */
+    private final int index;
```

Comment:
Do you think it would make more sense to include the use of the abstracted wrapper class `Index` here?

Reply:
Yes, that is a good suggestion! I have incorporated into my new commit.